clang/test/CodeGenCXX/cxx0x-initializer-scalars.cpp

```diff
@@ -5,3 +5,12 @@ void f()
   // CHECK: store i32 0
   int i{};
 }
+
+
+namespace GH116440 {
+void f() {
+  void{};
+  void();
+}
```

Comment:
Can you add check-lines to this one?

Reply:
Just the function header then `CHECK-NEXT: ret void` confirms that it actually does nothing/makes it through codegen without going sideways.

So:
```
// CHECK: define{{.*}} void @{{.*}}GH116440{{.*}}()
// CHECK-NEXT: {{.*}}:
// CHECK-NEXT: ret void
```

That should work on both windows and itanium manglers, though be more specific on the function name if this test uses a triple.